pettingzoo/sisl/multiwalker/multiwalker.py

```diff
@@ -239,3 +248,17 @@ def step(self, action):
 
         if self.render_mode == "human":
             self.render()
+
+    def state(self):
+        all_walker_obs = self.env.get_last_obs()
+        all_walker_obs = np.array(list(all_walker_obs.values())).flatten()
+        package_obs = np.array(
+            [
+                self.env.package.position.x,
+                self.env.package.position.y,
+                self.env.package.angle,
+            ]
+        )
+        global_state = np.concatenate((all_walker_obs, package_obs)).astype(np.float32)
```

Comment:
Makes sense to me at least, the two things that define the state are the package and the different walkers observations